### DISCUSSION TITLE
漢翔揪航太產業上下游，以AI解決實務挑戰

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#689] [General]
受到疫情影響，航運業不論是在海上的還是空中的，都受到大大的影響。在2019年全球航空業的景氣谷底，漢翔（民用飛機機體結構、次組合件之設計加工及代工製造，以及國際民用飛機之引擎零組件設計加工及代工製造等）不但不氣餒，反而訂下有關AI應用的目標。

在製程面，將AI導入熱壓爐的智慧監控，大幅改善生產品質和效率。
在複合材料的封膠工序，也以人機協作搭配AI視覺辨識，獲得更好的作業成果。
在營運面，則利用AI文字探勘，掌握全球各角落最即時的商情，了解客戶、同業對手、及業界動態，以快速反應迎戰激烈的市場競爭。

AI的應用真的很廣泛，各行各業各種技術都可能會用到，因此懂得在各個領域應用AI的人將會是有相當高的優勢。

文章連結：https://www.thenewslens.com/feature/aishow/177167

**Comment by Vincent20030314** [comment]
AI確實對很多產業有很大的幫助，成熟的AI就好比一個充足的勞動力。

**Comment by HsuChen88** [comment]
AI不論應用在哪個產業應該都能有很好的效果，大家都可以準備拿40個月的年終。

**Comment by yjpxi** [comment]
我覺得在製程方面可以利用AI視覺辨識來協助人機協作，能夠更精確地進行材料處理和檢查，減少人為錯誤，提升作業效果。
